private-templates-service/adaptivecards-templating-client/swagger.yaml

```diff
@@ -6,7 +6,7 @@ info:
   license:
     name: "Apache 2.0"
     url: "http://www.apache.org/licenses/LICENSE-2.0.html"
-host: "adaptivecms.azurewebsites.net"
+host: "localhost:5000"
```

Comment:
is this intentional

Reply:
Yep, we replace it in the pipeline depending on which env